styles/colors.css

```diff
@@ -15,15 +15,13 @@
   --activity-color-third: 25, 97, 39;
 }
 
-@media (prefers-color-scheme: light) {
-  :root {
-    --bg-color: 242 242 242;
-    --bg-color-danger: 255, 247, 247;
+[data-theme="light"] {
+  --bg-color: 242 242 242;
+  --bg-color-danger: 255, 247, 247;
 
-    --text-color-primary: 0 0 0;
-    --text-color-secondary: 85, 100, 125;
-    --text-color-third: 165, 180, 210;
+  --text-color-primary: 0 0 0;
+  --text-color-secondary: 85, 100, 125;
+  --text-color-third: 165, 180, 210;
```

Comment:
Let's add comma's to rgb colors `242 242 242` . Like `255, 247, 247`

Also in other examples with CSS variables